_source/_posts/2018-11-15-node-express-typescript.md

```diff
@@ -12,6 +12,8 @@ tweets:
 - "Build a @nodejs application using @typescriptlang and #expressjs"
 image: blog/node-express-typescript/node-express-typescript.jpg
 type: conversion
+changelog: 
+  - 2021-03-30: Updated to use CLI for creating Okta application.
```

Comment:
I've been adding changelogs when I updated libraries. Otherwise, I've left it out. If you leave it in, I'd recommend linking to this PR so people can see what changed.

```
You can see what's changed in [okta-blog#642](https://github.com/oktadeveloper/okta-blog/pull/642).
```